src/utilities/http.ts

```diff
@@ -8,5 +8,7 @@ export const HTTP_STATUS_CODES = {
   NOT_FOUND: 404,
   METHOD_NOT_ALLOWED: 405,
   CONFLICT: 409,
+  UNSUPPORTED_MEDIA_TYPE: 415,
+  FAILED_DEPENDENCY: 424,
```

Comment:
Looks like a bad merge. Are you sure this won't duplicate these lines when merged in `main` eventually?